src/pages/billing/CreateNewBillingProjectWizard.js

```diff
@@ -63,6 +64,7 @@ const CreateNewBillingProjectWizard = ({ onSuccess, billingAccounts, authorizeAn
     Utils.withBusyState(setIsBusy)
   )(async () => {
     try {
+      Ajax().Metrics.captureEvent(Events.billingProjectWizardStep4, { buttonName: 'Create Terra Billing Project' })
```

Comment:
So this one could be `billing:creation:gcpBillingProjectCreated`.

Reply:
Actually, remove this one because you put the logging in the `onSuccess` callback in List.js.